solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -0,0 +1,151 @@
+
+    Args:
+    costumer_id (string): costumer id.
+    amount_to_withdrawal (integer): amount the costumer wants to withdrawal.
+    costumers_dictionary (dictionary): dictionary of all costumers in
+                                       the bank.
+    """
+    balance_now = check_the_balance(costumer_id, costumers_dictionary)
+    if amount_to_withdrawal > 0:
+        if balance_now >= amount_to_withdrawal:
+            new_balance = balance_now - amount_to_withdrawal
+            costumers_dictionary[costumer_id][1] = str(new_balance)
+            return True
+    return False
+
+
+def cash_deposit(costumer_id, amount_to_deposit, costumers_dictionary):
+    """deposit to the bank some of money.
+
+    Args:
+    costumer_id (string): costumer id.
+    amount_to_deposit (integer): amount the costumer wants to deposit.
+    costumers_dictionary (dictionary): dictionary of all costumers in
+                                       the bank.
+    """
+    if amount_to_deposit > 0:
+        balance_now = check_the_balance(costumer_id, costumers_dictionary)
+        new_balance = balance_now + amount_to_deposit
+        costumers_dictionary[costumer_id][1] = str(new_balance)
+        return True
+    return False
+
+
+def change_password(costumer_id, new_password, costumers_dictionary):
+    """Changing the costumer's ATM password.
+
+    Args:
+    costumer_id (string): costumer id.
+    new_password (string): the new password the costumer wants to change to.
+    costumers_dictionary (dictionary): dictionary of all costumers in
+                                       the bank.
+    """
+    if len(new_password) == 4 and new_password.isdigit():
+        costumers_dictionary[costumer_id][0] = new_password
+        return True
+    return False
+
+
+def main():
+    costumers_dictionary = {}
+    try:
+        with open(sys.argv[ATM_FILE], 'r') as atm_file:
+            for line in atm_file:
+                costumers_dictionary[line.split()[0]] = line.split()[1:]
+
+    except FileNotFoundError:
+        print("Your file path is incorrect. Please check it.")
+
+    try:
+        print("welcome to bank!\n")
+        costumer_id = input("enter your id:\n")
+        while costumer_id != str(-1):
+            if len(costumer_id) == 9 and costumer_id.isdigit() and \
+                    costumers_dictionary.get(costumer_id):
+                print("for checking your balance press 1")
+                print("for cash withdrawal press 2")
+                print("for cash deposit press 3")
+                print("for change password press 4")
+                costumer_choice = int(input("pressing: \n"))
+
+                if costumer_choice == 1:
+                    print("your balance: ", end='')
+                    print(check_the_balance
+                          (costumer_id, costumers_dictionary))
+                    costumer_id = input("enter your id/ to stop and save"
+                                        " changes press (-1):\n")
```

Comment:
instead of writing the ID again to continue, why not:
"enter action 1-4, press -1 to exit..."